doc-content/enterprise-only/release-notes/rn-7.10-known-issues-ref.adoc

```diff
@@ -0,0 +1,187 @@
++
+You receive *Uncaught exception: Exception caught: (TypeError) : Cannot read property 'Tc' of undefined Caused by: (TypeError) : Cannot read property 'Tc' of undefined* exception.
+
+Workaround: None.
+
+.*Settings* panel is not updated when you make the changes in the the *DMN model* field [https://issues.redhat.com/browse/RHDM-1589[RHDM-1589]]
+
+Issue: In an existing DMN test scenario file, when you change the *DMN model* field, the related *DMN name* and *DMN namespace* read-only fields are not updated. The content of the *Settings* panel is out of sync.
+
+Steps to reproduce:
+
+. Open any existing test Scenario for DMN model.
+. Open the *Settings* panel.
+. Change the value of *DMN model* field.
+. Verify the values of *DMN name* and *DMN namespace* fields.
+
+Workaround: Close and reopen the asset.
+ifdef::PAM[]
+
+== Process Designer
+
+.JavaScript language in an *On Entry Action* causes an unexpected system error after changing node to *Multiple Instance* [https://issues.redhat.com/browse/RHPAM-3409[RHPAM-3409]]
+
+Issue: In the *Properties* panel, if the language is set to JavaScript in an *On Entry Action* property and you change the node to *Multiple Instance*, you receive an unexpected system error.
+
+Steps to reproduce:
+
+. Create a new business process.
+. Create a task that contains *Multiple Instance* property.
+. Enter any string to *On Entry Action* property.
+. Change the language to JavaScript.
+. Set the value of *Multiple Instance* property to `true`.
+
+Expected result: No errors occur either in user interface or server log.
+
+Actual result: You receive an unexpected system error.
+
+Workaround: None.
+
+.Service task thumbnails are large in documentations [https://issues.redhat.com/browse/RHPAM-2759[RHPAM-2759]]
+
+Issue: Service task thumbnails are large as compared to other nodes thumbnails in *Documentation*.
+
+Steps to reproduce:
+
+. Import the `thumbnails.bpmn` process attached in the https://issues.redhat.com/browse/RHPAM-2759[RHPAM-2759] issue.
+. Create a service task.
+. Close and reopen the process.
+. Open the *Documentation* tab and scroll down.
+
+Expected result: Service task thumbnails are of the same size as as compared to other nodes thumbnails.
+
+Actual result: Service task thumbnails are large as compared to other nodes thumbnails.
+
+Workaround: None.
+
+.Missing `structureRef` for message event when you create a process [https://issues.redhat.com/browse/RHPAM-3437[RHPAM-3437]]
+
+Issue: When you create a new process, the message event does not contain the `structureRef` in the generated BPMN file.
+
+Steps to reproduce:
+
+. Create a new process.
+. Add message start event and message end event.
+. Configure the message for the nodes.
+. Add a new process variable to the process.
+. Add a newly created process variable as an input or output of both the nodes.
++
+The generated BPMN file does not contain the `structureRef` for `endMessageType`, as per the following example:
+[source]
```

Comment:
I'd suggest aligning this source code. Add a `+` between the sentence and [source].